### PR TITLE
Not suggested for the second and subsequent Hashtags in the search word

### PR DESCRIPTION
**Pull request by tkhsknsk**
s-tip/stip-sns#106

### CODE REVIEW

File: src/templates/base.html [hunk 1]
```diff
@@ -65,6 +64,7 @@
                 <span id="search-button" class="input-group-btn">
                   <button type="submit" class="btn btn-default"><span class="glyphicon glyphicon-search"></span></button>
                 </span>
+                <div id="suggest" style="display:none;"></div>
```
Comment (ta-hiroki) [inline src/templates/base.html:67]: [Here is the CSS configuration](https://github.com/s-tip/stip-common/pull/73/files#diff-37c270a90f81e2757a9f2faf00f4832a38f181a995ddcd5b5abafabb7b2f85c3R116). Do you need `style="display:none;`?

Comment (tkhsknsk) [reply src/templates/base.html:67]: Thank you.
Go to CSS.
https://github.com/s-tip/stip-common/pull/73/files#diff-37c270a90f81e2757a9f2faf00f4832a38f181a995ddcd5b5abafabb7b2f85c3R108

Comment (tkhsknsk) [reply src/templates/base.html:67]: If you move it to CSS, it will not work, so please leave it as it is.

---

CSSに移動させると機能しなくなるのでそのままにさせてください。

Comment (tkhsknsk) [reply src/templates/base.html:67]: It is controlled by `suggest.js` and the official usage examples are written in HTML, so it should not be changed.

---

`suggest.js` で制御していること、公式の利用例でもHTMLに記述しているため、変更すべきでない。

**Review by ta-hiroki — APPROVED** [review]

**Pull request merged**